### PR TITLE
Fixed 1704 Fixing Preview for PC and Desktops

### LINKED ISSUE #1704: Fixing the Preview Badge Position for Desktops,Pc
**I'm submitting a ...**
- [x] Feature request 

**Current behaviour:**

Currently  Badge is fixed to left 
screenshot from 2018-09-14 21-43-44

So when  user scrolls he doesn't come to know about changes in badge , user as to go up to see changes,  instead if position remains fixed for card then user  can view changes.

Currently :
screenshot from 2018-09-15 00-14-26

**Expected behaviour:**

Similar to  right div which will provide preview:

image9

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes #1704  

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:
Working Demo :smile: 
ezgif com-video-to-gif 3

### CODE REVIEW

**Comment by sk9331657** [comment]
@ParthS007  @gabru-md  @abishekvashok  New update for Badge Preview is ready.  :smile: .

**Comment by sk9331657** [comment]
> Can you somehow avoid the initial scroll? Rest looks good on the preview. Looking at the diff

Scrolling Takes place only when create-badge loads.

File: frontend/app/components/create-badge.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import Component from '@ember/component';
+import Ember from 'ember';
+const EmberObject  = Ember;
+export default Component.extend({
+  didInsertElement() {
+    EmberObject.$(window).on('resize scroll', function(e) {
+      if (e.type === 'scroll' && this.scrollY >= 137 && this.window.innerWidth >= 764 && this.scrollY < 1278) {
+        this.document.getElementsByClassName('six wide column preview')[0].style.position = 'fixed';
```
Comment (gabru-md) [inline frontend/app/components/create-badge.js]: can we please avoid using vanilla javascript for ember components?
I've pointed it out previously as well. Can you make the changes please :smiley:

**Comment by sk9331657** [comment]
I will be fixing the bug by today night.  :smile:

**Comment by sk9331657** [comment]
@ParthS007  Done Changes. @gabru-md  Any changes required??

**Comment by sk9331657** [comment]
@abishekvashok @abishekvashok @gabru-md  Please review it.

File: frontend/app/components/create-badge.js (hunk 1 above)
Comment (sk9331657) [reply frontend/app/components/create-badge.js]: Is this fine now @gabru-md ??

**Comment by sk9331657** [comment]
@ParthS007  Can this be merged for now ?? I will implement best practices for future patches.

**Comment by ParthS007** [comment]
@sk9331657, Please made the requested changes after that It can be merged. :+1:

**Comment by sk9331657** [comment]
This is now here https://github.com/fossasia/badgeyay/pull/1733

**Pull request closed without merging**